example/controllers/index.js

```diff
@@ -0,0 +1,35 @@
+let ad;
+import {ADTYPES, AdView} from 'ti-admob';
+
+function doShowAd() {
+	if (!ad)
+		return;
+	if (ad.isLoaded()) {
+		ad.show();
+	} else {
+		console.warn('ad not yet loaded');
```

Comment:
Ti.API.warn? TBH, we should remove Ti.API.[info/warn/error/trace/debug] in favor of console.* in the future to be more Node-like

Reply:
@hansemannn console doesn't work in classic

Reply:
console does not work in classic?! Why that? Only with ES6?